lib/components/customs/CustomPicture.vue

```diff
@@ -117,6 +117,9 @@ export default {
       this.imageSources = this.preload;
       this.inProgress = false;
       this.$emit('load');
+      global.requestAnimationFrame(() => {
+        doPolyfill(this.$el, this.$refs.image);
```

Comment:
in the future we should split up the compontent into custom-picture and custom-image. so the refs part is not neccessary after the split.